NineChronicles.Headless/GraphTypes/TransactionHeadlessQuery.cs

```diff
@@ -0,0 +1,134 @@
+using System;
+using GraphQL;
+using GraphQL.Types;
+using NineChronicles.Headless;
+using NineChronicles.Headless.GraphTypes;
+using Libplanet.Blockchain;
+using Libplanet.Action;
+using Libplanet.Tx;
+using Libplanet;
+using Libplanet.Explorer.GraphTypes;
+using Nekoyume.Action;
+using NCAction = Libplanet.Action.PolymorphicAction<Nekoyume.Action.ActionBase>;
+using System.Text.Json;
+using System.Linq;
+using System.Collections.Generic;
+using Libplanet.Crypto;
+using Newtonsoft.Json.Linq;
+
+namespace NineChronicles.Headless.GraphTypes
+{
+    class TransactionHeadlessQuery : ObjectGraphType
```

Comment:
IMO, `TransactionHeadlessQuery` seems a little bit verbose. how about just `TransactionQuery`?

Reply:
As you suggested, we had named `TransactionQuery` but it has conflicted with `Libplanet.Explorer.Quries.TransactionQuery,<T>`. So we decided to use `TransactoinHeadlessQuery`. 😭 

Do you have any other idea to resolve the situation? 🤔

Reply:
I guess `TransactionQuery` and `TransactionQuery<T>` are different types even at the assembly level. (like `System.Collections.Generic.Dictioniary<K, V>` and `Bencodex.Types.Dictionary`). 

Can you let me know where and how the conflict occurs?

Reply:
You are right. The types are different in .NET type level. But GraphQL type doesn't allow it.

I saw these logs if I used `TransactionQuery` name:

```
System.InvalidOperationException: Unable to register GraphType 'NineChronicles.Headless.GraphTypes.TransactionQuery' wi
th the name 'TransactionQuery';
the name 'TransactionQuery' is already registered to 'Libplanet.Explorer.Queries.TransactionQuery`1[[Libplanet.Action.P
olymorphicAction`1[[Nekoyume.Action.ActionBase, Lib9c, Version=1.0.0.0, Culture=neutral, PublicKeyToken=null]], Libplan
et, Version=0.12.0.0, Culture=neutral, PublicKeyToken=null]]'.
   at GraphQL.Types.GraphTypesLookup.SetGraphType(String typeName, IGraphType type) in /_/src/GraphQL/Types/GraphTypesL
ookup.cs:line 593
   at GraphQL.Types.GraphTypesLookup.AddType(IGraphType type, TypeCollectionContext context) in /_/src/GraphQL/Types/Gr
aphTypesLookup.cs:line 274
   at GraphQL.Types.GraphTypesLookup.AddTypeWithLoopCheck(IGraphType resolvedType, TypeCollectionContext context, Type
namedType) in /_/src/GraphQL/Types/GraphTypesLookup.cs:line 414
   at GraphQL.Types.GraphTypesLookup.AddTypeIfNotRegistered(Type type, TypeCollectionContext context) in /_/src/GraphQL
/Types/GraphTypesLookup.cs:line 445
   at GraphQL.Types.GraphTypesLookup.HandleField(IComplexGraphType parentType, FieldType field, TypeCollectionContext c
ontext, Boolean applyNameConverter) in /_/src/GraphQL/Types/GraphTypesLookup.cs:line 365
   at GraphQL.Types.GraphTypesLookup.AddType(IGraphType type, TypeCollectionContext context) in /_/src/GraphQL/Types/Gr
aphTypesLookup.cs:line 279
   at GraphQL.Types.GraphTypesLookup.Create(IEnumerable`1 types, IEnumerable`1 directives, Func`2 resolveType, INameCon
verter nameConverter, Boolean seal) in /_/src/GraphQL/Types/GraphTypesLookup.cs:line 124
   at GraphQL.Types.Schema.CreateTypesLookup() in /_/src/GraphQL/Types/Schema.cs:line 311
   at System.Lazy`1.ViaFactory(LazyThreadSafetyMode mode)
   at System.Lazy`1.ExecutionAndPublication(LazyHelper executionAndPublication, Boolean useDefaultConstructor)
   at System.Lazy`1.CreateValue()
   at System.Lazy`1.get_Value()
   at GraphQL.Types.Schema.get_AllTypes() in /_/src/GraphQL/Types/Schema.cs:line 127
   at GraphQL.Instrumentation.FieldMiddlewareBuilder.ApplyTo(ISchema schema) in /_/src/GraphQL/Instrumentation/FieldMid
dlewareBuilder.cs:line 71
   at GraphQL.DocumentExecuter.ExecuteAsync(ExecutionOptions options) in /_/src/GraphQL/Execution/DocumentExecuter.cs:l
ine 75
```

Reply:
🆗